### ISSUE TITLE
Table of icons in create new view is not scrollable

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#6864]
**Scenario:**
1. Log in
2. Go to e.g. Companies
3. Click on number of rows, then "Add view" button"
4. Try to scroll table of available icons
![image](https://github.com/user-attachments/assets/623041cd-4d8b-4cd2-91ec-12c85c86772e)

**Actual:** Table of icons in create new view is not scrollable

**Expected:** Table of icons in create new view should be scrollable

**Comment by Faisal-imtiyaz123** [comment]
@BOHEUS It's because there is `max-height` set . This is related to #6105 . That PR will also solve it.

**Comment by FelixMalfait** [comment]
Closing in favor of #6105

**Comment by harshrajeevsingh** [comment]
Hey @BOHEUS @FelixMalfait, this issue should be reconsidered again. It's happening not because of ```max-height```, as mentioned by Faizal. The ```max-height```  property is already set as ```none```. 
It's happening because we are rendering a fixed number of icons, i.e., 25. 

If you want more clarification, please look at the screenshot. I increased the value of icons to 42. 
![Screenshot from 2024-10-17 20-26-45](https://github.com/user-attachments/assets/3262934e-4248-40fc-a5b1-25a581f3cd32)

**Comment by FelixMalfait** [comment]
@harshrajeevsingh good point! But there are so many icons / I'm afraid it might be complex to build something nice here. They'd need to be lazy loaded to avoid perf issues. Unless they are already loaded.. @lucasbordeau what do you think, would that be complex?

**Comment by harshrajeevsingh** [comment]
@FelixMalfait Yes, there are more than 4k icons. The performance issue can be fixed using a virtualization library like react-window, but the User Experience would be awful. We should take a different approach.

**Comment by Bonapara** [comment]
We can do infinite scroll with lazy loading + change the search to `Search 4000+ icons` and still limit the height to 25 icons max.

**Comment by BOHEUS** [comment]
@lucasbordeau done
